ocs_ci/ocs/platform_nodes.py

```diff
@@ -32,7 +32,7 @@ class PlatformNodesFactory:
 
     """
     def __init__(self):
-        self.cls_map = {'AWS': AWSNodes, 'vsphere': VMWareNodes, 'aws': AWSNodes, 'baremetal': NodesBase}
+        self.cls_map = {'AWS': AWSNodes, 'azure': AZURENodes 'vsphere': VMWareNodes, 'aws': AWSNodes, 'baremetal': NodesBase}
```

Comment:
comma "," missing and azure should be using NodeBase class, as AZURENodes class is missing